### DISCUSSION TITLE
Sound overload

### DISCUSSION THREAD

**Discussion by curiousvlxd** [kyleneideck/BackgroundMusic#625] [Ideas]
Please implement sound overload protection. The user should have a checkbox/setting by default for protecting himself by not setting more volume than Apple provides by default, speakers could break.

**Comment by kyleneideck** [comment]
Background Music plays audio to your speakers the same way any other app does. If your speakers need overload protection in software, it'll be in your speaker's driver and won't get bypassed.

The drivers for built-in speakers on MacBooks do that IIRC. You can even tell it's still running because it uses a decent amount of CPU time.
